modules/Bio/Vega/DBSQL/GeneAdaptor.pm

```diff
@@ -934,6 +936,28 @@ sub fetch_all_genes_on_reference_slice {
     return $transformed_genes;
 }
 
+sub add_persistent_attributes {
```

Comment:
@mira13 Can we have this subroutine in a separate module ? I can code is duplicated in all the modules